core/sqf/src/seatrans/tm/hbasetmlib2/src/main/java/org/trafodion/dtm/HBaseTxClient.java

```diff
@@ -364,9 +335,15 @@ public void nodeUp(int nodeID) throws IOException {
            return;
        }
        rt.stopThread();
-       try {
-           rt.join();
-       } catch (Exception e) { LOG.warn("Problem while waiting for the recovery thread to stop for node ID: " + nodeID); }
+       boolean loopBack = false;
+       do {
+          try {
+             rt.join();
```

Comment:
Should there be a loopBack = false after rt.join in the case where an exception occurred but then the operation succeeded?